ava-test/cat.js

```diff
@@ -0,0 +1,61 @@
+import fs from 'fs';
+
+import test from 'ava';
+
+import shell from '..';
+
+test.beforeEach(() => {
+  shell.config.silent = true;
+});
```

Comment:
Doing it each time should be harmless. But since it slows things down, it should be fine to move it out to just once before.